docs/scheduler.md

```diff
@@ -0,0 +1,60 @@
+# Edge Scheduler
+
+This is a component in an architecture not yet shared in detail. It is
+an early milestone in edge multicluster, achieving partial
+functionality and implemented in a simple crude approach that is
+layered on top of transparent multicluster. In this approach the
+edge-mc layer maintains a kcp workspace, called a mailbox, per
+selected edge destination per EdgePlacement object. Then TMC syncs
+between the mailbox workspaces and the edge destinations.
+
+## Development Status
+
+Currently the code is just the barest start.  
+
+There is a start on the scheduler.  It only creates an informer on
+EdgePlacement objects and logs what the informer reports.  This
+controller is currently a stand-alone process. No leader election.
+Not containerized.
+
+Requires commit 4506fdc06406 (from Jan 19) of kcp-dev/kcp.
+
+## Build
+
+Build with ordinary go commands.
+
+### NOTE WELL
+
+Because https://github.com/kcp-dev/edge-mc/issues/136 is not solved,
+the CRDs have been hacked by hand to state that they are NOT
```

Comment:
That issue has now been solved by https://github.com/kcp-dev/edge-mc/pull/151 and closed